examples/Lock-In Amplifier Measurements/LI5660-Data1.py

```diff
@@ -0,0 +1,58 @@
+from PyQt5 import QtWidgets, QtCore
+from pyqtgraph import PlotWidget, plot
+import pyqtgraph as pg
+import sys  # We need sys so that we can pass argv to QApplication
+import os
+
+import pyvisa
+rm = pyvisa.ResourceManager()
+instrument = rm.list_resources()
+from pymeasure.instruments.nf import LI5660
+
+class MainWindow(QtWidgets.QMainWindow):
+    
+    def __init__(self, *args, **kwargs):
+        super(MainWindow, self).__init__(*args, **kwargs)
+
+        self.nf = LI5660(instrument[0])
+
+        self.graphWidget = pg.PlotWidget()
+        self.setCentralWidget(self.graphWidget)
+
+        # self.x = list(range(100))  # 100 time points
+        # self.y = [randint(0,100) for _ in range(100)]  # 100 data points
+
+        self.fetchData = self.nf.fetch_data
```

Comment:
You don't use the `self.fetchData` later on (in stead you use the `self.nf.fetch-data`, which is indeed equally fine, later on) so there is no need to define this I think.

Reply:
Noted with thanks.